src/templates/drf/form-field/form-field.html

```diff
@@ -1,15 +1,16 @@
-<div class="control-group" ng-class="{error: field.errors.length}">
-    <label class="control-label" ng-if="label" bind-html-compile="label"></label>
+<div class="control-group" ng-class="{error: field.errors.length, required: field.required}">
+    <label class="control-label">
+        <span ng-if="label" bind-html-compile="label"></span>
+        <span ng-if="field.required" class="asterisk">*</span>
```

Comment:
I think there would probably only be an asterisk if there's a label, maybe it should be inside the first span?